setup.py

```diff
@@ -20,6 +20,7 @@
         'cryptography>=1.1.1, <2.0.0',
         'flask-appbuilder>=1.6.0, <2.0.0',
         'Flask-BabelPkg==0.9.6',
+        'Flask-Cors>=2.0.0',
```

Comment:
I would rather not have this as a hard dependency for the vast majority. I'd propose removing it or using `extra_requires`.